.github/ISSUE_TEMPLATE/NEW_RELEASE.md

```diff
@@ -49,13 +50,11 @@ Please do not remove items from the checklist
 - [ ] Run the [SBOM action](https://github.com/kubernetes-sigs/kueue/actions/workflows/sbom.yaml) to generate the SBOM and add it to the release.
 - [ ] For major or minor releases, merge the `main` branch into the `website` branch to publish the updated documentation.
 - [ ] Send an announcement email to `sig-scheduling@kubernetes.io` and `wg-batch@kubernetes.io` with the subject `[ANNOUNCE] kueue $VERSION is released`.   <!--Link: example https://groups.google.com/a/kubernetes.io/g/wg-batch/c/-gZOrSnwDV4 -->
-- [ ] Update the below files with respective values in `main` branch :
-  - Latest version in `README.md`
-  - Latest version in `cmd/experimental/kjobctl/docs/installation.md`
-  - Release notes in the `CHANGELOG`
-  - `version` in `site/hugo.toml`
-  - `appVersion` in `charts/kueue/Chart.yaml`
-  - `SECURITY-INSIGHTS.yaml` values by running `make update-security-insights GIT_TAG=$VERSION`
```

Comment:
I guess that this is still needed, right?

Reply:
oh yes, SECURITY_INSIGHTS is not fixed by `prepare-release-branch`. I suppose we could add a script to automate it, but maybe it's not worth yet.

Reply:
Right, added it back

Reply:
I think so too.